ui/components/component-library/avatar-account/avatar-account.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import Jazzicon from '../../ui/jazzicon/jazzicon.component';
+import BlockieIdenticon from '../../ui/identicon/blockieIdenticon/blockieIdenticon.component';
+import { BaseAvatar } from '../base-avatar';
+
+import { SIZES } from '../../../helpers/constants/design-system';
+
+const Diameters = {
+  xs: '16',
+  sm: '24',
+  md: '32',
+  lg: '40',
+  xl: '48',
+};
+
+export const types = ['Jazzicon', 'Blockie'];
```

Comment:
Wondering if we can move these into a file called `avatar-account.constants.js` and export them from there so they can be used in this file and the stories? Similar to `ui/components/component-library/icon/icon.constants.js`? I think we can make that a standardization and it also follows what mobile is doing. Could we also make them all uppercase consts like in the design-system?

```
export const DIAMETERS = {
  XS: '16',
  SM: '24',
  MD: '32',
  LG: '40',
  XL: '48',
};

export const TYPES = {
  JAZZICON: 'jazzicon',
  BLOCKIES: 'blockies',
};
```